CRM/Case/BAO/Case.php

```diff
@@ -1369,6 +1369,8 @@ public static function sendActivityCopy($clientId, $activityId, $contacts, $atta
     if ($caseId) {
       $activityInfo['fields'][] = array('label' => 'Case ID', 'type' => 'String', 'value' => $caseId);
     }
+    $allActivityTypes = CRM_Core_PseudoConstant::activityType(TRUE, TRUE);
+    $tplParams['activityTypeName'] = $allActivityTypes[$activityTypeId];
```

Comment:
I think it would be better to use the `CRM_Core_PseudoConstant::getLabel()` function (example of that below in the diff of `CRM_Case_XMLProcessor_Report`).

Reply:
@colemanw thanks for the review, we'll update the PR in the next few days.